src/Hosting/Hosting/src/Internal/WebHost.cs

```diff
@@ -317,7 +317,7 @@ public async Task StopAsync(CancellationToken cancellationToken = default)
             }
             _stopped = true;
 
-            _logger?.Shutdown();
+            _logger.Shutdown();
```

Comment:
Revert. It's possible (though not a good idea) to call Stop without first calling Start.

Reply:
What if I assign `NullLogger` to `_logger` in constructor?

Reply:
That could work.